config.json

```diff
@@ -788,7 +788,12 @@
         "name": "High Scores",
         "uuid": "f5af74f7-259c-48e6-828c-bcd98717d8dd",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": [
+          "arrays",
+          "array-transformations",
+          "rest-and-spread",
+          "classes"
+        ],
```

Comment:
Looks like the main branch is not merged to your branch as I thought this is merged by now. :thinking:

Reply:
Oh, I think I have figured out the mistake, it is on my side. I had checked out to this branch from the high scores branch instead of first checking out to the main branch. I am not sure what to do.

Reply:
Here is how I fix these issues:
1. Go to the main branch `git checkout main` and pull the latest changes `git pull upstream main` (this is also available via the GitHub UI).
2. Switch back to your branch and merge the main branch `git merge main`.
3. Push the changes. Then the part that is already in the main branch is not in the diff of the PR anymore.

I did this for you here so I can merge this now.

Reply:
Oh cool. I will make sure to do this if it happens next time. Thanks for the reply 😃